src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java

```diff
@@ -24,6 +25,7 @@
     public Envelope(
         @JsonProperty(value = "id", required = true) String id,
         @JsonProperty(value = "case_ref", required = true) String caseRef,
+        @JsonProperty("previous_service_case_ref") String legacyCaseRef,
```

Comment:
are there any changes made in the processor yet?
If not, maybe we could have `legacy_case_ref` in json?

Reply:
The processor is already passing this field. I'll change the name as the last thing in this story.